lib/prepare.js

```diff
@@ -160,8 +160,20 @@ class Prepare{
             }
 
             for (const directory of directories) {
-                if(process.env.WORKER_TEST_MODE){
-                    await fse.copy(directory.out, "/out", copyOptions);
+                if (process.env.WORKER_TEST_MODE) {
+                    await fse.copy(directory.out, "/out", {
+                        // Make sure symlinks are copied as binaries and not symlinks
+                        dereference: true,
+                        // ensure files can be read by host system by running chmod before copy
+                        filter: src => {
+                            if (directory.postprocessing && src.startsWith(directory.postprocessing)) {
+                                return false;
+                            }
```

Comment:
looks like you can just add these lines to `copyOptions` and it would have the same effect with less repetitive code: https://github.com/adobe/asset-compute-pipeline/blob/8a224e13b044a227cce8963ed704c2eb3b54eb64/lib/prepare.js#L149

Reply:
The problem is that you need to have `directory` which is inside the loop, but. copyOption is used outside of the loop as well.